MC-2875 Vector Collections (#333)
hazelcast/management-center-docs: docs/modules/data-structures/pages/vector-collection.adoc

> = Vector Collections

Reviewer: ```suggestion
= Vector Collections
:description: Learn how to use and monitor vector collections within Management Center.
```

|Puts
|Number of put operations received by the member.

|Put All
|Number of put all operations received by the member.

|Put All Entries
|Number of put all entries received by the member.

|Deletes
|Number of delete operations received by the member.

|Removes
|Number of remove operations received by the member.

|Clears
|Number of clear operations received by the member.

|Optimizes
|Number of optimize operations received by the member.

|Sizes
|Number of size operations received by the member.

|Searches
|Number of search operations received by the member.

|Search Results
|Number of search results.

|===

You can ascend or descend the order of the listings by clicking on
the column headings.

[[vector-collection-throughput]]**Vector Collection Throughput Table** provides information about the operations
(get, put, remove) performed on each member in the vector collection, as shown below.

image:ROOT:VectorCollectionThroughputDataTable.png[Vector Collection Throughput Data Table]

From left to right, this table lists the following:

.Vector Collection Throughput Statistics fields
[cols="20%s,80%a"]
|===
|Field|Description

|Members
|IP address and port of the member.

> |Gets/s
> |Number of get operations per second on the member.
> 
> |Puts/s
> |Number of put operations per second on the member.
> 
> |Put Alls/s
> |Number of put all operations per second on the member.
> 
> |Removes/s
> |Number of remove operations per second on the member.
> 
> |Sets/s
> |Number of local set operations per second on the member.
> 
> |Deletes/s
> |Number of delete operations per second on the member.
> 
> |Clears/s
> |Number of clear operations per second on the member.
> 
> |Searches/s
> |Number of search operations per second on the member.
> 
> |Optimizes/s
> |Number of optimize operations per second on the member.
> 
> |Sizes/s
> |Number of size operations per second on the member.

Reviewer: ```suggestion
|Gets
|Number of get operations per second on the member.

|Puts
|Number of put operations per second on the member.

|Put Alls
|Number of put all operations per second on the member.

|Removes
|Number of remove operations per second on the member.

|Sets
|Number of local set operations per second on the member.

|Deletes
|Number of delete operations per second on the member.

|Clears
|Number of clear operations per second on the member.

|Searches
|Number of search operations per second on the member.

|Optimizes
|Number of optimize operations per second on the member.

|Sizes
|Number of size operations per second on the member.
```

Reviewer 2: `/s` means per second and these fields are for operations per second, so `/s` should stay

|Average latency of delete operations on the member.

|Avg Clear Latency
|Average latency of clear operations on the member.

|Avg Search Latency
|Average latency of search operations on the member.

|Avg Optimize Latency
|Average latency of optimize operations on the member.

|Avg Size Latency
|Average latency of size operations on the member.

|Max Avg Get Latency
|Maximum Average latency of get operations on the member.

|Max Avg Put Latency
|Maximum Average latency of put operations on the member.

|Max Avg Put All Latency
|Maximum Average latency of put all operations on the member.

|Max Avg Remove Latency
|Maximum Average latency of remove operations on the member.

|Max Avg Set Latency
|Maximum Average latency of set operations on the member.

|Max Avg Delete Latency
|Maximum Average latency of delete operations on the member.

|Max Avg Clear Latency
|Maximum Average latency of clear operations on the member.

|Max Avg Search Latency
|Maximum Average latency of search operations on the member.

|Max Avg Optimize Latency
|Maximum Average latency of optimize operations on the member.

|Max Avg Size Latency
|Maximum Average latency of size operations on the member.

|===

> You can select the time period in the combo box placed in the top
> right corner of the window, for which the table data will be
> shown. Available values are **Since Beginning**, **Last Minute**,
> **Last 10 Minutes** and **Last 1 Hour**.

Reviewer: ```suggestion
You can select the time period for the table using the combo box in the top
right corner of the window:
* Since Beginning
* Last Minute
* Last 10 Minutes
* Last 1 Hour
```

> To ascend or descend the order of the listings, click on the
> column headings.

Reviewer: ```suggestion
To change the order of entries to ascending or descending, click on
the appropriate column heading.
```
